data/README.md

```diff
@@ -53,6 +53,7 @@
 | incorrect_map_schema.parquet | Contains a Map schema without explicitly required keys, produced by Presto. See [note](#incorrect-map-schema) |
 | column_chunk_key_value_metadata.parquet | two INT32 columns, one with column chunk key-value metadata {"foo": "bar", "thisiskeywithoutvalue": null} note that the second key "thisiskeywithoutvalue", does not have a value, but the value can be mapped to an empty string "" when read depending on the client |
 | sorting_columns.parquet | INT64 and BYTE_ARRAY columns with first column with nulls first and descending, second column with nulls last and ascending. This file contains two row groups with same data and sorting columns. |
+| old_list_structure.parquet | Single LIST<LIST<INT32>> column with legacy two-level list structure |
```

Comment:
Can you document the file like you did in the PR description?

Reply:
Added. Please take a look again. Thanks!